prometheus-connector/README.md

```diff
@@ -0,0 +1,189 @@
+## Flink Prometheus connector (sink)
+
+Implementation of the Prometheus sink connector for DataStream API.
+
+The sink writes to Prometheus using the Remote-Write interface, based on [Remote-Write specifications version 1.0](https://prometheus.io/docs/concepts/remote_write_spec/)
+
+### Guarantees and input restrictions
+
+Due to the strict [ordering](https://prometheus.io/docs/concepts/remote_write_spec/#ordering) and [format](https://prometheus.io/docs/concepts/remote_write_spec/#labels) requirements
+of Prometheus Remote-Write, the sink guarantees that input data are written to Prometheus only if input data are in order and well-formed.
+
+For efficiency, the connector does not do any validation.
+If input is out of order or malformed, the write request is rejected by Prometheus and data is discarded by the sink.
```

Comment:
We can mention our sink configurations

Reply:
Configuration is extensively documented below, in the README

Reply:
Sorry, understood the point. Adding an explanation